docs/examples/camera_display.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    mask = tailcuts_clean(geom, image, 10, 5)
</pre><p>Single threshold?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/ctapipe/pull/2266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
What do you mean?

Reply:
the comment says "simpliefied cleaning, single threshold" but what really is done is using tailcuts with two thresholds